gemfiles/rails5_0_0.gemfile

```diff
@@ -1,10 +1,14 @@
 source "https://rubygems.org"
 
+gem "rails", "5.0.0"
```

Comment:
Why did you need to add a gem such as Rails?

Reply:
I asked to be listed in the DESCRIPTION.

Reply:
https://github.com/mixigroup/activerecord-turntable/pull/18#issuecomment-1610825952